int16 controlId
string name
int16 numVal